cli/src/commands/mod.rs

```diff
@@ -70,7 +72,21 @@ impl From<&ExitCode> for i32 {
             ExitCode::JsError => 16,
             ExitCode::ConfirmationFailed => 17,
             ExitCode::FailedThresholds => 100,
+            ExitCode::SandboxStart => 117,
+            ExitCode::SandboxStartCollision => 118,
             ExitCode::Custom(code) => *code,
         }
     }
 }
+
+impl PartialEq<i32> for ExitCode {
+    fn eq(&self, code: &i32) -> bool {
+        i32::from(self).eq(code)
+    }
+}
+
+impl PartialEq<ExitCode> for i32 {
+    fn eq(&self, code: &ExitCode) -> bool {
+        self.eq(&i32::from(code))
+    }
+}
```

Comment:
This sets up a non-transitive equality relationship.

Specifically:
* `ExitCode::Custom(117) == 117`
* `117 == ExitCode::SandboxStart`
* `ExitCode::Custom(117) != ExitCode::SandboxStart`

This seems odd to me. But I guess it's probably fine here because of the types.

Reply:
I didn't think of custom. This is actually not fine according to Rust's official guidelines.